src/webui/www/private/rename_files.html

```diff
@@ -61,7 +61,7 @@
 
             // Register keyboard events to modal window
             // https://github.com/qbittorrent/qBittorrent/pull/18687#discussion_r1135045726
-            const keyboard = new Keyboard({
+            new Keyboard({
```

Comment:
This is the last `new Keyboard()` instance. I failed to replace it due to the new code couldn't catch `escape` key event. The event was sent to the top level `window` instead of the `windowEl` (local window element).
Anyway, we can revise it later.